executor/serverless/serverless.go

```diff
@@ -33,9 +33,37 @@ const (
 	sdInitPrefix = "sdinit-"
 )
 
-// checks if launcher update is required
-func checkLauncherUpdate(serviceClient *awsAPI, launcherVersion string) bool {
+// awsRegionMap for region short names
+var awsRegionMap = map[string]string{
+	"north":     "n",
+	"west":      "w",
+	"northeast": "nw",
```

Comment:
not sure if this is a typo
```suggestion
	"northwest": "nw",
```

Reply:
oh it should be `northeast: "ne"`